OracleWebLogic/samples/12213-webtier-apache/README.md

```diff
@@ -0,0 +1,43 @@
+Example of Apache Web Server with WebLogic 12.2.1.3.0 Plugin
+=====
+This sample shows how to run a load balancer for a WebLogic cluster, from inside a container in the same network as the WLS Cluster.
+
+# How to build image
+Download file **Oracle WebLogic Server Proxy Plugins 12.2.1** [fmw_12.2.1.3.0_wlsplugins_Disk1_1of1.zip](http://www.oracle.com/technetwork/middleware/webtier/downloads/index-jsp-156711.html) (see .download file)
+
+You may build this image with:
+
+        $ docker build -t 12213-apache .
+
+or
+
+        $ sh build.sh
+
+Oracle developers can use build_internal.sh, which set up HTTP/HTTPS proxies.
```

Comment:
If you need to set up HTTP/HTTPS proxies, build the image using script build_internal.sh